Dockerfile

```diff
@@ -1,10 +1,20 @@
 FROM registry.access.redhat.com/ubi8/python-39
 
 ENV PATH "$PATH:/opt/app-root/packages/bin/"
-ENV PYTHONPATH=/opt/app-root/src/src:/opt/app-root/packages
+ENV PYTHONPATH=/opt/app-root/src/rhub/src:/opt/app-root/packages
+
+RUN dnf install openssh-clients; \
+    dnf clean all
+
+RUN mkdir -p ~/.ssh \
+    && chmod 700 ~/.ssh \
+    && printf "Host *.redhat.com\n\tStrictHostKeyChecking no\n" > ~/.ssh/config \
+    && chmod 600 ~/.ssh/config
+
+WORKDIR /opt/app-root/src/rhub/
 
 COPY ./requirements.txt ./requirements.txt
-RUN pip3 install pip-tools && pip-sync --pip-args '-t ../packages'
+RUN pip3 install pip-tools && pip-sync --pip-args '-t ../../packages --upgrade'
```

Comment:
We don't want unexpected updates and broken dependencies. If you want upgrade packages run `pip-compile -U` and commit the new requirements txt.

Reply:
Thanks for pointing this. I had an issue during the development, but I couldn't reproduce it anymore.
And I'm not sure why this fixed the issue, as all the versions are pinned on `requirements.txt`.